### PR TITLE
Cleanup Integration Test Import

### LINKED ISSUE #1118: Change MANTA as KMA import in integration tests
https://github.com/Manta-Network/Manta/blob/55f15d96c2056d379f24223e52db4904387e9d42/runtime/integration-tests/src/integrations_mock/mod.rs#L54

This line is confusing and could cause bad tests to be run due to the non-intuitiveness of it. We should use UNIT or some other moniker

### PR DESCRIPTION
**Pull request by ferrell-code**
## Description

close: #1118 

---

Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
- [x] Connected to an issue with discussion and accepted design using zenhub "Connect issue" button below
- [x] Added **one** label out of the `L-` group to this PR
- [x] Added **one or more** labels from the `A-` and `C-` groups to this PR
- [x] Explicitly labelled `A-calamari` and/or `A-manta` if your changes are meant for/impact either of these (CI depends on it)
- [x] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functionality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.

### CODE REVIEW

File: runtime/integration-tests/src/integrations_mock/mod.rs [hunk 1]
```diff
@@ -48,10 +48,11 @@ cfg_if::cfg_if! {
         type RuntimeAssetConfig = calamari_runtime::assets_config::CalamariAssetConfig;
         type RuntimeConcreteFungibleLedger =
             calamari_runtime::assets_config::CalamariConcreteFungibleLedger;
+        const UNIT: Balance = KMA;
     } else {
         pub mod test_manta;
         use manta_runtime::{
-            currency::MANTA as KMA,
+            currency::MANTA,
```
Comment (Garandor) [inline runtime/integration-tests/src/integrations_mock/mod.rs:55]: why not just `use currency::MANTA as UNIT` ?

Comment (ferrell-code) [reply runtime/integration-tests/src/integrations_mock/mod.rs:55]: Well the thought is that then `KMA` can be used in calamari specific tests and `MANTA` can be used in manta tests and then `UNIT` is used for common tests.

But maybe just changing all tests to use `UNIT` is cleaner. I am cool w either way

Comment (Garandor) [reply runtime/integration-tests/src/integrations_mock/mod.rs:55]: I'd personally prefer `UNIT` across all tests, but that's just my preference. Ok leaving as it is

**Review by Garandor — APPROVED** [review]

**Review by Apokalip — APPROVED** [review]
LGTM

**Pull request merged**